packages/legend-extension-dsl-data-space/src/components/DSL_DataSpace_LegendApplicationPlugin.tsx

```diff
@@ -204,4 +206,34 @@ export class DSL_DataSpace_LegendApplicationPlugin
       },
     ];
   }
+
+  getExtraQueryBuilderFunctionHelper(
```

Comment:
this should also be an abstract method on QueryBuilderState and we implement in DataSpaceQueryBuilderState